src/main/java/com/launchdarkly/client/LoggerFactory.java

```diff
@@ -0,0 +1,17 @@
+package com.launchdarkly.client;
+
+import java.util.concurrent.atomic.AtomicReference;
+import org.slf4j.Logger;
+
+public class LoggerFactory {
+
+    private static AtomicReference<ILoggerFactory> factory = new AtomicReference<>();
+
+    public static void install(ILoggerFactory factory) {
```

Comment:
In many cases loggers where instantiated in classes that did not refrence the config so I've left them as is. Where possible it now goes to the LDConfig to get a logger instance.

I couldn't get gradle to build locally so you should double check this all works before merging :/ sorry for the hassle